config/manager/manager.yaml

```diff
@@ -33,6 +33,7 @@ spec:
         - /usr/local/bin/manager
         args:
         - --leader-elect
+        - '-v=2'
```

Comment:
any reason to add it here?

Reply:
Please see the review history. This is the default verbosity for the manager. We will update the Troubleshooting Guides in the operator section to explain to administrators how to increase the verbosity level.